hm_pyhelper/hardware_definitions.py

```diff
@@ -449,6 +493,8 @@ def is_raspberry_pi() -> bool:
     # Nebra Indoor Hotspot
     'DIY-RAK2287': {
         'FRIENDLY': 'DIY RAK2247/RAK2287 HAT',
+        'CPU_ARCH': 'armv7hf',
```

Comment:
I wonder whether these DIY ones should be defined in some other way or split out based on actual arch or something.

Or is there a way we can keep this blank and pass it on where needed.

Needs some more thought probably